libs/financial-aid/shared/src/lib/utils.ts

```diff
@@ -25,3 +26,24 @@ export const formatPhoneNumber = (phoneNumber: string) => {
 
 export const formatNationalId = (nationalId: string) =>
   insertAt(nationalId.replace('-', ''), '-', 6) || '-'
+
+export const sanitizeNationalId = (nationalId: string) =>
+  nationalId.replace(/[^0-9]/g, '')
+
+export const isEmailValid = (emailAddress?: string) => {
+  if (emailAddress) {
+    const regex = /\S+@\S+\.\S+/
```

Comment:
Wondering if we should have a more strict regex check, we did for example this one earlier this year: https://github.com/island-is/island.is/blob/d923e58a8e8d8744401ce098e49be73379a00427/libs/application/templates/family-matters/core/src/utils/dataSchema.ts